prqlc/bindings/prqlc-python/src/lib.rs

```diff
@@ -50,17 +50,45 @@ pub fn rq_to_sql(rq_json: &str, options: Option<CompileOptions>) -> PyResult<Str
         .map_err(|err| (PyErr::new::<exceptions::PyValueError, _>(err.to_json())))
 }
 
+mod debug {
+    use super::*;
+
+    #[pyfunction]
+    pub fn prql_lineage(prql_query: &str) -> PyResult<String> {
```

Comment:
If you have the patience we could also add a quick snapshot tests to these; there's an example with `compile` at the bottom. But not strictly necessary.